test/modules/TestResultCoordinator/Reports/DeploymentTestReport.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Reports
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    /// <summary>
+    /// This is a counting report to show deployment test result, e.g. expect and actual number of deployments.
+    /// </summary>
+    class DeploymentTestReport : TestResultReportBase
+    {
+        public DeploymentTestReport(
+            string trackingId,
+            string expectedSource,
+            string actualSource,
+            string resultType,
+            ulong totalExpectedDeployments,
+            ulong totalActualDeployments,
+            ulong totalMatchedDeployments,
+            Option<TestOperationResult> lastActualDeploymentTestResult,
```

Comment:
Why is the lastActualDeploymentTestResult needed?

Reply:
If there is any mismatched result, this lastActualDeploymentTestResult can help for investigation.